testing/simapp/test_helpers.go

```diff
@@ -197,7 +233,7 @@ func SetupWithGenesisAccounts(genAccs []authtypes.GenesisAccount, balances ...ba
 	app.InitChain(
 		abci.RequestInitChain{
 			Validators:      []abci.ValidatorUpdate{},
-			ConsensusParams: DefaultConsensusParams,
+			ConsensusParams: simtestutil.DefaultConsensusParams,
```

Comment:
are these parameters the same as what was defined at the top of the file?

Reply:
They are identical apart from the max gas config. Previously, `MaxGas: 2000000` now `MaxGas: 100_000_000`

I don't see any issues with this